src/net.cpp

```diff
@@ -731,6 +897,95 @@ void V1TransportSerializer::prepareForTransport(CSerializedNetMsg& msg, std::vec
     CVectorWriter{SER_NETWORK, INIT_PROTO_VERSION, header, 0, hdr};
 }
 
+void V2TransportSerializer::prepareForTransport(CSerializedNetMsg& msg, std::vector<unsigned char>& header)
+{
+    size_t serialized_command_size = 1; // short-IDs are 1 byte
+    const uint8_t cmd_short_id = GetShortCommandIDFromCommand(msg.command);
+    if (cmd_short_id == 0) {
+        // message command without an assigned short-ID
+        assert(msg.command.size() <= 12);
+        // encode as varstr, max 12 chars
```

Comment:
9f81125 could replace the int value of `12` here and line 809 above with an explicit `CMD_MAX_CHARS_SIZE` (or similar) int static constant

Reply:
Good point. Changed.